tests/drivers/redis/QueueTest.php

```diff
@@ -137,4 +138,41 @@ protected function tearDown()
         $this->getQueue()->redis->flushdb();
         parent::tearDown();
     }
+
+    public function testMoveExpired()
```

Comment:
It is not clear from the code what this test does. Worth either making naming cleaner or adding some description.